active_directory_ldap/komand_active_directory_ldap/connection/schema.py

```diff
@@ -19,7 +19,7 @@ class ConnectionSchema(komand.Input):
     "host": {
       "type": "string",
       "title": "Host",
-      "description": "Server Host, e.g. ldap://example.com. Must use either ldap:// or ldaps:// for SSL prefix",
+      "description": "Server Host, e.g. example.com for SSL prefix",
```

Comment:
Looks like this didn't get regenerated after your last commit. 

Did the validator catch this?